scss/_patterns_image.scss

```diff
@@ -1,6 +1,37 @@
 @import 'settings';
 
 @mixin vf-p-image {
+  %vf-image-container--common {
+    img {
+      background-size: contain;
+    }
+  }
+
+  [class^='p-image-container'] {
+    @extend %vf-image-container--common;
+  }
+
+  .p-image-container--16-9 {
+    aspect-ratio: 16/9;
+  }
+  .p-image-container--3-2 {
+    aspect-ratio: 3/2;
+  }
+  .p-image-container--2-3 {
+    aspect-ratio: 2/3;
+  }
+  .p-image-container--cinematic {
+    aspect-ratio: 2.4/1;
+  }
+  .p-image-container--square {
+    aspect-ratio: 1/1;
+  }
+
+  .p-image-container--highlighted-background {
```

Comment:
This also kinda throws a wrench in the starts-with selector above, since it would create a selector containing duplicate styles from the `common` placeholder.